result-app-service.yml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: result-service
   labels:
     name: result-service
-    app: demo-voting-app
+    app: demo-result-app
```

Comment:
The app name is the name of your app and in this case it's a voting app. It should remain the same for all configs (pods or service)

Reply:
Thank you!